docs/source/markdown/podman.1.md

```diff
@@ -95,10 +106,16 @@ Storage driver option, Default storage driver options are configured in /etc/con
 
 **--syslog**
 
-output logging information to syslog as well as the console
+Output logging information to syslog as well as the console.
 
 On remote clients, logging is directed to the file ~/.config/containers/podman.log
 
+**--tmpdir**
+
+Path to the tmp directory, for libpod runtime content.
+
+NOTE --tmpdir is not used for the temporary storage of downloaded images.  Use the environment variable `TMPDIR` to change the temporary storage location of downloaded container images. Podman defaults to use `/var/tmp`.
```

Comment:
Shouldn’t this added only _after_ it is true? See https://bugzilla.redhat.com/show_bug.cgi?id=1769918 .

Reply:
Yes I guess so.  The question is how quickly can we move...